README.md

```diff
@@ -110,6 +110,13 @@ To enable/disable the autostart service perform the following steps:
   $ chkconfig --del autostartliota
 ```
 
+### On systemd based systems:
+
+```bash
+  $ systemctl enable liota.socket
```

Comment:
Should it be liotad.service & liotad.socket?

Also,  I cloned your branch and will try out this commands on Fedora VM.  What is the prime job of this commands? Also, I think if liota daemon gets started then we won`t require liotad.socket to be started explicitly.